pyiron/base/generic/parameters.py

```diff
@@ -1004,11 +1004,4 @@ def _find_line(self, key_name):
         elif len(i_line_lst) == 1:
             return i_line_lst[0], multi_word_lst[0]
         else:
-            for i in i_line_lst:
```

Comment:
Is there a specific reason why you remove these lines? I guess they are quite helpful for debugging, so we should at least add them to the logger.

Reply:
Fixed here https://github.com/pyiron/pyiron/pull/437/commits/7a860d969037f8d2537967d3d75245e09ba4baa0. Is this what you mean?